src/Components/FormCreation/AddCardComponent.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlus, faLayerGroup } from "@fortawesome/free-solid-svg-icons";
+
+const useStyles = makeStyles({
+  collapse: {
+    marginBottom: 16
+  },
+  content: {
+    marginLeft: 10,
+    padding: 0
+  },
+  expandOpen: {
+    transform: "rotate(180deg)"
+  },
+  root: () => ({
```

Comment:
Is there a reason this a function? (not that it is bad, just curious)  

I normally do it with just `{ }`

Reply:
I had copied that from a different class. It's not needed here so I'll remove it when I do the css refactoring. I think for that class it had some variables being used in the css so it was needed? Not sure.